Kickstarter-iOS/Views/PledgeScreenCTAContainerView.swift

```diff
@@ -0,0 +1,199 @@
+import KsApi
+import Library
+import PassKit
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Button {
+    static let minHeight: CGFloat = 48.0
+  }
+}
+
+final class PledgeScreenCTAContainerView: UIView {
```

Comment:
We don't refer to a `PledgeScreen` anywhere else in the code so I would go with `PledgeView` here to avoid confusion.